WARNING:root:Warning information
ERROR:root:Error information
CRITICAL:root:Critical information
INFO:root:A new  Log came
INFO:root:Log is recorded
INFO:root:A new  Log came
ERROR:root:division by zero
Traceback (most recent call last):
  File "C:\Users\user787\PycharmProjects\team-6\level5\logtest.py", line 7, in <module>
    print(n1/n2)
ZeroDivisionError: division by zero
INFO:root:Log is recorded
